compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/OwnedLayerManager.kt

```diff
@@ -40,4 +40,6 @@ internal interface OwnedLayerManager {
     fun notifyLayerIsDirty(layer: OwnedLayer, isDirty: Boolean)
 
     fun invalidate()
+
+    fun voteFrameRate(frameRate: Float)
```

Comment:
Nitpick

```suggestion
    fun voteFrameRate(frameRate: Float) = Unit
```
to avoid changes in tests.

🤔 probably we can do the same for existing methods to reduce useless overrides in tests. Super minor, feel free to ignore